stable/prometheus-blackbox-exporter/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Prometheus Blackbox Exporter
 name: prometheus-blackbox-exporter
-version: 0.2.0
+version: 0.3.0
```

since this chart is in the stable folder lets bump to 1.0.0
```suggestion
version: 1.0.0
```

---

this makes sense, however since this PR doesn't introduce major features or introduce any breaking changes that would warrant a major version bump under SemVer, might that be better as a separate PR?

---

(i don't have a super clear sense of what the culture is like here and how picky people are about git history, if it's easier to just put this in the PR i can do so)